### ISSUE TITLE
Issue with subseconds

### ISSUE THREAD

**Issue by andrewkendall** [janfri/multi_exiftool#7]
I'm currently tracking an issue with subseconds which looks like it was introduced in 0.6.0.

I'm seeing that when I call

`image_exif.SubSecDateTimeOriginal`

0.5.0 and below produce the result:

2012:07:13 19:36:01.32

0.6.0 and above produces:

2012-07-13 19:36:01 +0100

Calling subsec on the times from 0.6.0 and above results in 0.

This seems to be the case for all the files I've tested with.

I've attached a sample image:
![sample](https://user-images.githubusercontent.com/11718/43167026-4ac6f92c-8f90-11e8-86ab-8b16b8a1ae02.jpg)

This is the test code to reproduce:

```ruby
#!/usr/bin/ruby

gem 'multi_exiftool', '0.9.0'

require 'multi_exiftool'
multi_exiftool_reader = MultiExiftool::Reader.new
multi_exiftool_reader.filenames = Dir['*.jpg']
multi_exiftool_reader_results = multi_exiftool_reader.read
image_exif = multi_exiftool_reader_results.first
puts image_exif.SubSecDateTimeOriginal
```

**Comment by janfri** [comment]
I'm just back from holiday. Therefore my answer is a little bit late.

Yes this was a bug introduced in version 0.6.0. The intention was to handle timestamps with fraction of seconds and converting them also to Time objects.

It is fixed in version 0.9.1:
```ruby
MultiExiftool.read('43167026-4ac6f92c-8f90-11e8-86ab-8b16b8a1ae02.jpg')[0][0].SubSecDateTimeOriginal.strftime('%Y-%m-%d %H:%M:%S.%2N')
# => "2012-07-13 19:36:01.32" 
```

Thanks for your analyze and reporting.

Btw: Great photo.

**Comment by andrewkendall** [comment]
Perfect. No problem at all, that was still fast despite of the holiday. I hope that you had a relaxing time :)

I've just tested the latest version in my code and it looks to be working great again.

Many thanks for all the hard work on MultiExiftool, it's really appreciated.

Glad you liked the picture! It's an older one from my archives, Lana Del Rey at Latitude Festival.
